src/Tes/Utilities/ConnectionStringUtility.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Text;
+
+namespace Tes.Utilities
+{
+    public class ConnectionStringUtility
+    {
+        public string GetPostgresConnectionString(
```

Comment:
I think passing the options object rather than the individual values is a more maintainable approach. PostgreSqlOptions can be added/removed/changed without changing the signature of the method.